src/core/TrustManager.sol

```diff
@@ -28,21 +28,17 @@ abstract contract TrustManager is IRegistry {
     /**
      * @inheritdoc IERC7484
      */
-    function trustAttesters(
-        uint8 threshold,
-        address[] memory attesters // deliberately using memory to allow sorting and uniquifying
-    )
-        external
-    {
+    function trustAttesters(uint8 threshold, address[] calldata attesters) external {
         uint256 attestersLength = attesters.length;
 
-        // sort attesters and remove duplicates
-        attesters.sort();
-        attesters.uniquifySorted();
```

Comment:
@kopy-kat  please check this. I changed it from sorting the array in memory, to just validation that the array is already sorted. this will save us some gas

Reply:
looks good to me